Mage/Logger/Writers/ConsoleWriter.cs

```diff
@@ -56,7 +56,7 @@ public class ConsoleWriter {
 		Logger.logEmitter.on ("critical", (object sender, LogEntry logEntry) => {
 			Debug.LogError (makeLogString("critical", logEntry.context, logEntry.message));
 			if (logEntry.data != null) {
-				if (logEntry.data is Exception) {
+				if (logEntry.data is Exception && (logEntry.data as Exception).StackTrace != null) {
```

Comment:
yeah, since it doesn't have a stack it will automatically `toString()` it's message